src/index.html

```diff
@@ -7,8 +7,71 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link 
+      rel="preconnect" 
+      href="https://fonts.gstatic.com" 
+      crossorigin="link"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#" class="header__logoLink">
+        <img 
+          class="header__logoImg" 
+          src="images/logo.png" 
+          alt="MOYO Logo"
+        >
+      </a>
+      <nav class="nav header__nav">
+        <ul class="nav__linksBar">
+          <li class="nav__listItem">
+            <a href="#apple" class="nav__link is-active">
+              APPLE
+            </a>
+          </li>
+          <li class="nav__listItem">
+            <a href="#samsung" class="nav__link">
+              SAMSUNG
+            </a>
+          </li>
+          <li class="nav__listItem">
+            <a href="#smartphones" class="nav__link">
+              SMARTPHONES
+            </a>
+          </li>
```

Comment:
```suggestion
          <li class="nav__listItem">
            <a href="#apple" class="nav__link is-active">
              APPLE
            </a>
          </li>
          
          <li class="nav__listItem">
            <a href="#samsung" class="nav__link">
              SAMSUNG
            </a>
          </li>
          
          <li class="nav__listItem">
            <a href="#smartphones" class="nav__link">
              SMARTPHONES
            </a>
          </li>
```

Reply:
All comments corrected. Thank you for your revision!